### DISCUSSION TITLE
tui-multi-select limit the maximum number of selected options

### DISCUSSION THREAD

**Discussion by antoxers71** [taiga-family/taiga-ui#1695] [Q&A] [answered]
HI! Is it possible to limit the maximum number of selected options for a tui-multi-select?

**Comment by waterplea** [comment, marked as answer]
You can just disable options once control value length exceeds threshold. If you use data list wrapper, there's an input for disabled item handler

**Reply by antoxers71** [reply]
Thank you!)
